billboard.py

```diff
@@ -346,8 +392,19 @@ def getEntryAttr(selector):
             if artist == "":
                 title, artist = artist, title
 
-            # TODO: Parse the image
-            image = None
+            # get image data from artists list
+            artistData = list(filter(lambda x: x['name'] == artist,
```

Comment:
Sounds like `artists` should be a dict.

Reply:
Although, what happens if an artist appears multiple times on a chart with different albums? Does `titleImage` work correctly?

Reply:
Yes, you're right. I'll fix that